docs/guides/access-groups.mdx

```diff
@@ -105,45 +139,58 @@ When you create this access group, include the following policies.  You can copy
-      - For **Service access**, select one of the following:
-         - **Reader**, if the user only needs to view (but not send) jobs.
-         - **Writer**, if the user needs to send, update, and delete jobs.
-         - **Manager**, if in addition to Writer and Reader actions, the user needs to update the instance `max cost limit`.
-         - Your own custom role (optional), defined on the [Roles](https://cloud.ibm.com/iam/roles) page.
-      - For **Platform access**, select **Viewer**.
-   - **Conditions (optional)**. Click **Review**.
+      - For **Service access**, select  **Manager**.
+      - For **Platform access**, select **Administrator**.
+
+   At the bottom, click **Add**. You should see the policy on the right-hand panel. Click **Assign** at the bottom of that panel.
+
+You have successfully created an access group with one policy.  Next, create a second policy for the same instance.
+
+1. In the same access group, click the **Access** tab, then click **Assign access**.
+1. In the **Create policy** page that opens, define these elements:
+   - **Service** - Select **All IAM Account Management services**. Click **Next**.
+   - **Roles and actions** - For Platform access, select **Administrator**, Click **Next**.
+   At the bottom, click **Add**, then click **Assign**.
+
+Create a third policy for the same instance.
 
-   At the bottom, click **Add**. You should see the policy on the right-hand panel. Click **Assign** at the bottom of the right-hand panel. You have successfully created an access group for your instance.
-1. On the left panel in the **Manage identities** section, click **Users**. Then, at the top right click **Invite users**.
-   - Enter the email addresses of your users separated by commas, spaces, or line breaks.
-   - Select **Access groups** as the way to assign access.
-   - In the access groups table, select the one you just created and click **Add**. You will see the Access summary on the right-hand panel. Click **Invite** at the bottom of the Access summary panel.
+1. In the same access group, click the **Access** tab, then click **Assign access**.
+1. In the **Create policy** page that opens, define these elements:
+   - **Service** - Select **All Account Management services**. Click **Next**.
+   - **Roles and actions** - For Platform access, select **Viewer**, Click **Next**.
+   At the bottom, click **Add**, then click **Assign**.
 
-Users will receive an email with an invitation to use this account with the defined access to the instance. After they accept the invitation, they will see a new account in their account switcher at the top of the IBM Quantum Platform page. After selecting the new account, they will see the shared instance.
+Finally, invite users to the access group.  You can do this from the access group's **Users** page, or by using the IBM Quantum Platform [Access management](https://quantum.cloud.ibm.com/admin/access-groups) page.
```

Comment:
In the **Users** page from the Access Group you can only add users already invited or part of the account, you can not invite users from there, you have to invite them first through the **Users** page in the account where you can also select which access groups they will have when joining the account. Same in the platform.

The URL has been updated to https://quantum.cloud.ibm.com/access-groups